src/Routes.js

```diff
@@ -95,6 +96,7 @@ const Routes = (props) => (
       rolesAllowed={[USER_ROLES.all]} />
     <AuthenticatedRoute path="/tos_acceptance" component={TermsOfServiceAcceptance} props={props} rolesAllowed={[USER_ROLES.all]} />
     {(props.env === 'local' || props.env === 'dev') && <AuthenticatedRoute path="/data_submission_form" component={DataSubmissionForm} props={props} rolesAllowed={[USER_ROLES.all]} />}
+    <AuthenticatedRoute path="/translate" component={Translator} props={props} rolesAllowed={[USER_ROLES.researcher]}/>
```

Comment:
Since this is a non-DUOS experiment, enable it when `env === local`

Reply:
I think Gabe is very hopeful to show this off in conjunction with https://github.com/DataBiosphere/consent-ontology/pull/671 once that is approved

Reply:
So then should we just limit it to just `dev` and `staging` for now to make it demoable?

Reply:
Seems reasonable to me!

Reply:
ok! updated!